app/javascript/components/record-list/components/date-column/component.spec.js

```diff
@@ -0,0 +1,27 @@
+import { fromJS } from "immutable";
+
+import { mountedComponent, screen } from "../../../../test-utils";
+
+import DateColumn from "./component";
+
+describe("<DateColumn />", () => {
+  const props = {
+    value: "2020-04-150T14:04",
+    rowAvailable: true,
+    wrapper: null,
+    valueWithTime: false
+  };
+
+  it("renders a <DateColumn />", () => {
+    mountedComponent(<DateColumn {...props} />, fromJS({}));
+    expect(screen.queryByText(/2020-04-150T14:04/i)).not.toBeInTheDocument();
```

Comment:
I believe the date should be rendered because this test made sure the component was rendered.